dotcom-rendering/docs/development/ab-testing-in-dcr.md

```diff
@@ -52,3 +52,22 @@ const variantFromRunnable =
     AB Test
 </div>;
 ```
+
+## Server-side A/B tests
+
+In order to set up a server-side test in DCR, follow steps 1-4 outlined in the `frontend` [documentation](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/01-ab-testing.md#write-a-server-side-test).
+
+On the live website, [Fastly](https://github.com/guardian/fastly-edge-cache/blob/main/theguardiancom/src/main/resources/varnish21/ab-tests.vcl) automatically assigns users to buckets. You can force yourself into a test on your local machine by following these steps:
```

Comment:
Should we do deep linking to private repos from public repositories?

Reply:
Good spot, will remove the link!